sds_data_manager/utils/dependencies.json

```diff
@@ -0,0 +1,5 @@
+{
+    "l1a_Codice": ["l0_Codice"],
+    "l1b_Codice": ["l1a_Codice", "l1a_Mag"],
+    "l1c_Codice": ["l1b_Codice", "l2_Ultra", "spice"]
```

Comment:
Is this intended for all dependencies eventually? I'm a little worried this file will get massively out of hand. Also, as an imap specific piece of information, it might not belong in this repository. I'm thinking more and more this should eventually become a database. As for right now for the initial pass, this is totally cool, but I could see it getting out of hand...

Reply:
This is for the s3 bucket data triggers only. Calibration checks will happen in the lambas. I see it as being static and not complicated. But if it does get that way then we can look at the information going into a database. Or it could go into the database regardless of how complicated it is if it helps to keep us organized. Good for now at least.

Reply:
Ah yes, I understand. Ok, that is probably alright for now, although I agree with you that we can keep an eye on it and decide if it's overly complicated. It shouldn't affect this PR either way, though.